### PR TITLE
GljJ6ZKC [WIP] Guidance on handling the identity dataset

### PR DESCRIPTION
**Pull request by bravegrape**
### Background

We should offer guidance for what services should be doing if they need to do matching with the identity dataset they receive.

### Content plan

- What is matching?
- What to consider when matching?
- How to test other identities
- Any eIDAS implications?
- Test with different identities:
  - levels of assurance
  - test your matching strategy
    - different user attribtues (variety of names)
    - historical attributes

### CODE REVIEW

File: source/get-started-with-vsp/handle-identity-dataset/index.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: Handle the identity datset
```
Comment (jakubmiarka) [inline source/get-started-with-vsp/handle-identity-dataset/index.html.md.erb]: dat**a**set

File: source/get-started-with-vsp/handle-identity-dataset/index.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+weight: 30
+---
+
+# Handling the identity datset
```
Comment (alan-gds) [inline source/get-started-with-vsp/handle-identity-dataset/index.html.md.erb]: Dataset, again.

File: source/get-started-with-vsp/handle-identity-dataset/index.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+This page offers guidance on things you should consider if you need to link a user's verified identity to information your service already holds.
+
+## The identity dataset
+
+When a user's identity is successfully verified, your service receives an identity dataset which contains verified information about that user including their:
+
+- name
+- address
+- date of birth
+
```
Comment (alan-gds) [inline source/get-started-with-vsp/handle-identity-dataset/index.html.md.erb]: Are these definitely guaranteed to be present for eIDAS?  I was under the impression address was optional.

File: source/get-started-with-vsp/handle-identity-dataset/index.html.md.erb [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+If you need to check whether the identity dataset can be linked to information held by your service, you will need to implement a process called 'matching'. For more detailed information about setting up matching, refer to the [matching guidance section][matching].
+
+### Handling European identities
+
+Intro and link to eIDAS section: https://www.docs.verify.service.gov.uk/legacy/matching-guidance/matching-dataset/#pid-and-matching-dataset
+
+## Test your matching implementation
+
+You should test that your matching implementation correctly handles the [level of assurance][loa] of your service. If you have an LOA2 service, you must not accept LOA2 identities. However, as an LOA1 service, you can accept LOA2 identities.
```
Comment (alan-gds) [inline source/get-started-with-vsp/handle-identity-dataset/index.html.md.erb]: > If you have an LOA2 service, you must not accept LOA2 identities

Surely the latter should be LOA1 identities?

**Comment by bravegrape** [comment]
[VSP docs: handling the identity dataset](https://trello.com/c/GljJ6ZKC/127-vsp-docs-handling-the-identity-dataset)

**Comment by owenblacker** [comment]
We have decommissioned these docs today; this PR is no longer pertinent

**Pull request closed without merging**